src/sources/opentelemetry/tests.rs

```diff
@@ -0,0 +1,142 @@
+#[cfg(test)]
+mod tests {
```

Comment:
I think you can just remove the `mod tests { }` and move the cfg attr up into `mod.rs`.

Reply:
Ah - missed in the copy-pasting

Reply:
addressed in [56a888a](https://github.com/vectordotdev/vector/pull/13658/commits/56a888ab1d44498d9f8b7cfce3604d443c3dc6d3)